jwst/combine_1d/tests/test_combine1d.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from numpy.testing import assert_allclose
+
+from ... master_background import create_master_bkg
+from ...combine_1d import combine1d
```

Comment:
Is there a need for consistency as to whether there's a whitespace after "..." in a relative import like this?

Reply:
I would have thought it was required to not have a space, i.e. `from ...master_background` is explicitly preferred. I'm curious now though for myself, did we specify somewhere if the jwst package would use all relative explicit, or absolute imports?

Reply:
We didn't officially specify it anywhere, but it was agreed that we'd use relative explicit, as opposed to things like ``from jwst.master_background``, in order to avoid issues due to folks possibly having their own "jwst" defined in their environment (either intentionally or not). At least that's what I recall from 5 years ago.

Reply:
Here is an example from PEP 328 (I left out the comments because they're confusing):

    from . import D
    from .. import E                
    from ..F import G

I've seen other examples that are consistent with this.  That is, use a space if ".." (or whatever) just identifies a directory from which you want to import a module (D or E in the above examples).  Do not use a space between the last dot if what follows is a module name in the indicated directory (module F in .., and module F contains G).

Reply:
This file will be removed before the PR is merged. The goal is to replace `OutputSpectrumModel.to_pixel()` with a call to the `OutputSpectrumModel.wcs.inverse`. This test is meant to show that the two functions give the same results. 

otherwise :+1: for consistency in whatever form.